acceptable/openapi.py

```diff
@@ -21,17 +22,41 @@ def _to_dict(source: Any):
         for item in source:
             source_list.append(_to_dict(item))
         return source_list
+    elif hasattr(source, "__dict__"):
+        source_dict = {}
+        for key, value in source.__dict__.items():
+            source_dict[key] = _to_dict(value)
+        return source_dict
     else:
         return source
 
 
-def dump(metadata, stream):
+@dataclass
+class OasInfo(object):
+    description: str = ""
+    version: str = ""
+    title: str = ""
+    tags: list = field(default_factory=lambda: [])
+    contact: dict = field(default_factory=lambda: {"name": "", "email": ""})
+
+
+@dataclass
+class OasRoot31(object):
+    openapi: str = "3.1.0"
+    info: OasInfo = OasInfo()
+    servers: dict = field(default_factory=lambda: {})
+    paths: dict = field(default_factory=lambda: {})
+    components_schemas: dict = field(default_factory=lambda: {})
+
+
+def dump(metadata: APIMetadata, stream):
```

Comment:
Maybe make `stream=None` by default, to avoid awkward calls such as `dump(metadata, None)`.